client/my-sites/upgrades/domain-management/dns/dns-list.jsx

```diff
@@ -43,7 +43,11 @@ const DnsList = React.createClass( {
 	addDns: function( record ) {
 		addDnsAction( this.props.selectedDomainName, record, ( error ) => {
 			if ( error ) {
-				notices.error( error.message );
+				notices.error( error.message || this.translate( 'The DNS record could not be restored.' ) );
+			} else {
+				this.props.successNotice( this.translate( 'The DNS record has been restored.' ), {
+					duration: 5000
```

Comment:
@umurkontaci @klimeryk could you add a couple screenshots of what these notices are in the context of the DNS record? That would help frame the discussion.

The history of notices was like this:

* First we had a notices-list that rendered notices in a specific DOM element. It was wonky.
* Then we created a better simple `Notice` component to be used in place of notices-list. These notices are inline, managed by your component tree, and more contextual.
* Finally, a re-architecture and redesign of notices-list into a notices redux store rendered within a global-notices component. To be used with judgement.

In most every case we should be using inline notices where the state is handled by your components. There's no visual inconsistency since `<Notice>` handles the display. You can work more easily with these notices, specially when it comes to event handling.

Global notices were designed to contain sparser information, they are rendered as a single line pill, and it should be used for things that are so significant we want to make them global, and so clear to the user they don't need UI context. (Example: "your connection dropped off".)

====

In this case it seems inline notices would be better, since you can render them next to the record you are modifying giving better context.